lms/djangoapps/grades/tests/test_new.py

```diff
@@ -573,25 +582,34 @@ def test_course_grade_logging(self):
             course_id=self.course.id,
             enabled_for_course=True
         ):
-            with patch('lms.djangoapps.grades.new.course_grade.log') as log_mock:
-                # TODO: once merged with the "glue code" PR, update expected logging to include the relevant new info
+            with patch('lms.djangoapps.grades.new.course_grade.log') as log_mock:# the course grade has not been created, so we expect each grade to be created
```

Comment:
Note: The mock is created only once within this test method.  But this method gets the course grade multiple times, without resetting the mock.  So it's theoretically possible that the logs are actually being created by a previous call in the method.